CHANGELOG.md

```diff
@@ -68,6 +68,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Security
 
 ## [Unreleased 2.x]
+- Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
```

Comment:
These changes are on the 2.6 branch, right? Why are they going back into the changelog as unreleased?

Reply:
We are planning to release experimental version of extensibility in 2.8/2.9. If you think we should keep these changes in 2.6 release note, I'm fine with that.

Reply:
Let me know if this PR needs to be closed

Reply:
Well, this change is being released in 2.6 right? If this is worthy of being detailed in the release notes it should stay in. If this is just one incremental piece of a larger feature to be announced in a future release then maybe we can just remove it?

Reply:
This is an incremental change of a larger feature for the future release. Removing it sounds good to me for 2.6.

Reply:
Sounds good to me. Worth keeping in mind for all the incremental PRs that are yet to come as well :)